docs/topics/settings.rst

```diff
@@ -286,9 +288,20 @@ DEPTH_PRIORITY
 
 Default: ``0``
 
-An integer that is used to adjust the request priority based on its depth.
+Scope: ``scrapy.spidermiddlewares.depth.DepthMiddleware``
+
+An integer that is used to adjust the request priority based on its depth:
+
+- **a positive value will decrease the priority**
```

Comment:
I always had troubles reading scrapy docs when priorities are explained :) Sometimes a larger value means 'more priority', sometimes it means 'later in a queue', this is confusing.